app/assets/javascripts/ckeditor/plugins/cortex_media_insert/plugin.js

```diff
@@ -0,0 +1,34 @@
+(function (global) {
```

Comment:
Since `this` in this case refers to he window object, and the  `media_select_defer` is globally accessible then you should probably just use the window keyword.